GPUSimulators/Common.py

```diff
@@ -510,7 +518,7 @@ def __del__(self, *args):
     """
     Enables downloading data from GPU to Python
     """
-    def download(self, stream, cpu_data=None, async=False, extent=None):
+    def download(self, stream, cpu_data=None, asynch=False, extent=None):
```

Comment:
Don't change this. please leave as async.

Reply:
This change is necessary for newer versions of Python, see https://github.com/inducer/pycuda/issues/166